src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person:
+  { name,
+    age,
+    partnerName,
+    sex,
+    isMarried } }) => (
+      <section className="Person">
+        <h2 className="Person__name">
+          {`My name is ${name}`}
+        </h2>
+        { age !== undefined
+          ? (
+            <p className="Person__age">
+              {`I am ${age}`}
+            </p>
+          ) : (
+            <p className="Person__age" />
+          )
+        }
```

Comment:
You should NOT render the `<p className="Person__age">` if there is no age, even empty